test/k8sT/manifests/lrp-test.yaml

```diff
@@ -0,0 +1,136 @@
+SrcIP=10.80.0.9 DstIP=10.80.0.2
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: k8s1-backend
```

Comment:
The port names don't match with the ones specified in the yaml, is that intentional? Maybe this is the remote be? On a related note, does it make sense to name the pods as local-backend and remote-backend?

Reply:
> The port names don't match with the ones specified in the yaml

I'm a bit confused, which port name are you referring to?
 
> does it make sense to name the pods as local-backend and remote-backend

Since "local" and "remote" are relative to the requester, I think maybe we should keep the name generic here?

Reply:
Great catch, done :)